libats-logging/src/main/resources/logback-libats.xml

```diff
@@ -24,7 +24,7 @@
     <logger name="com.zaxxer.hikari" level="${HIKARI_LOG_LEVEL:-INHERITED}" />
     <logger name="slick.backend.DatabaseComponent.action" level="${SLICK_LOG_LEVEL:-INHERITED}" />
     <logger name="com.advancedtelematic" level="${ATS_LOG_LEVEL:-INHERITED}" />
-    <logger name="com.advancedtelematic.libats.messaging.kafka" level="${ATS_KAFKA_LOG_LEVEL:-INHERITED}" />
+    <logger name="com.advancedtelematic.libats.messaging" level="${ATS_MESSAGING_LOG_LEVEL:-INHERITED}" />
```

Comment:
Why this change? `.kafka` is there because of https://github.com/advancedtelematic/libats/blob/7e461829fca65a11bccc4d84f1947fa6351995b3/libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/kafka/KafkaClient.scala#L85

Reply:
Am I wrong assuming that this will handle logs under both `com.advancedtelematic.libats.messaging` and `com.advancedtelematic.libats.messaging.kafka` ?

Reply:
Yeah but I mean why not have `ATS_KAFKA_LOG_LEVEL` anymore?

Reply:
Ah so you'd rather have one logger for each. I just thought we didn't wouldn't be interested in that level of control, but yes sure, will update.

Reply:
I don't have a preference but I wanted to know why the change

Reply:
If you mean the change in the variable name it's just for consistency with the package. I did a `ack ATS_KAFKA_LOG_LEVEL` at `ota-charts` and found no usage so I figured changing the variable name wouldn't have any implications.

Reply:
Ah! Ok I thought ATS_MESSAGING_* already existed. ok ok